testData/solutions/solutionsDescription/com_microsoft_windows_closedCaptions.md

```diff
@@ -0,0 +1,82 @@
+# Windows Display Language
+
+## Details
+
+* __Name__: Windows Closed Captions
+* __Id__: com.microsoft.windows.closedCaptions
+* __Platform__: Windows
+* __Contact__: JJ <jj@raisingthefloor.org>
+
+## Description
+
+Sets several options about how captions should be displayed by media players. This options are:
+
+* BackgroundColor: Specifies which background color is going to be used for captions.
+
+  Possible values are:
+   - [ Default, White, Black, Red, Green, Blue, Yellow, Magenta, Cyan ].
+
+* BackgroundOpacity: Captions are displayed in a virtual text box in the media player, originally this box
+    is transparent. This property allows the user to change the transparency policy for that box.
+
+    Possible values are:
+    - [ Default, Opaque, Translucent, Semitransparent, Transparent ].
+
+* CaptionColor: Determines the color in which captions are going to be draw.
+
+    Possible values are:
+    - [ Default, White, Black, Red, Green, Blue, Yellow, Magenta, Cyan ].
+
+* CaptionEdgeEffect: Adds effects to the captions edge, currently this effects are not visible in the
+    "Movies" application from Windows 10.
+
+    Possible values are:
+    - [ Default, None, Raised, Depressed, Uniform, Drop shadow ]
+
+* CaptionFontStyle: Changes the font in which captions are displayed.
+
+    Possible values are:
+    - [ Default, Mono serif, Proportional serif, Mono sans serif, Proportional sans serif, Casual, Cursive, Small caps ].
+
+* CaptionOpacity: Determines the transparency of the displayed captions.
+
+    Possible values are:
+    - [ Default, Opaque, Translucent, Semitransparent, Transparent ].
+
+* CaptionSize: Determines the size of the displayed captions.
+
+    Possible values are:
+    - [ Default, 50%, 100%, 150%, 200%" ]
+
+
+The following options dims video content using a virtual background to improve subtitle reading,
+ by default, this virtual background is transparent.
+
+* RegionColor: Determines the color of the region used to dim video content.
+
+    Possible values are:
+    - [ Default, White, Black, Red, Green, Blue, Yellow, Magenta, Cyan ].
+
+* RegionOpacity: Determines the opacity of the region used to dim video content.
+
+    Possible values are:
+    - [ Default, Opaque, Translucent, Semitransparent, Transparent ].
+
+    NOTE: Remind that if the region opacity is "Opaque" will be impossible to see the video.
+
+## Testing
+
+The following test users make use of the closed captions setting:
+    - Catherine
+
+The following steps allows checking the actual changes in the subtitles:
+
+- Have a video with available subtitles tracks.
+- Loging in the system with a user using this feature.
+- Open the video with the windows player "Movies".
+- Select a subtitle track in the ones available for the video.
+- See the subtitles displayed with the selected options.
+
+## Limitations
+
+* Currently we are not able to trigger the subtitles because the system setting intended for that task isn't honored.

```

Comment:
Is there a link to any discussion or bug reports about this limitation?

Reply:
Yes, we had a discussion about it, as it was a major concern about this functionality. But there is no bug reports about the limitation filed. It looks like it's the intended behavior for the Windows 10 apps, to not honor that setting and give the user the election of selecting their preferred subtitle track. But anyway I can create a bug report and link to the issue, or simply add a not to the issue description, so we don't forget about it.

Reply:
Please file report and add link here - cheers!

Reply:
Done. Sadly, there is no public site for reports, Microsoft has closed it, and since middle of 2018, the unique place in which you can file those reports is through the "Feedback Hub" a Windows 10 UWP app, and the provided feedback is tied to the personal account of the user. I have provided a screenshot in JIRA with the information contained in the provided feeback.
https://issues.gpii.net/browse/GPII-2750